### PR TITLE
modify dockerfile: support cuda11 and delete gcc8.2 in cpu version

### PR DESCRIPTION
**Pull request by pangyoki**
### PR types

Others

### PR changes

Others

### Describe

ATT

### CODE REVIEW

File: tools/dockerfile/ubuntu16_dev.sh [hunk 1]
```diff
@@ -32,11 +32,15 @@ function ref_whl(){
...
 
-  if [[ ${gcc_version} == "8.2.0" ]];then
+  if [[ ${WITH_GPU} != "ON" ]]; then
+    ref_gcc = ""
+  elif [[ ${gcc_version} == "8.2.0" ]];then
     ref_gcc=_gcc8.2
   fi
 
-  if [[ ${ref_CUDA_MAJOR} == "10" ]];then
+  if [[ ${ref_CUDA_MAJOR} == "11" ]];then
```
Comment (tianshuo78520a) [inline tools/dockerfile/ubuntu16_dev.sh]: 需要改成11.0

**Review by tianshuo78520a — APPROVED** [review]
LGTM

File: tools/dockerfile/ubuntu16_dev.sh (hunk 1 above)
Comment (pangyoki) [reply tools/dockerfile/ubuntu16_dev.sh]: done

**Comment by tianshuo78520a** [comment]
LGTM

**Review by zhiqiu — APPROVED** [review]

**Pull request merged**